From: Andrii Nakryiko <andriin@fb.com>
Date: Wed, 9 Oct 2019 21:25:34 -0700
Subject: scripts/bpf: Fix xdp_md forward declaration typo
Patch-mainline: v5.5-rc1
Git-commit: e0b68fb186b251374adbd870f99b1ecea236e770
References: bsc#1155518

Fix typo in struct xpd_md, generated from bpf_helpers_doc.py, which is
causing compilation warnings for programs using bpf_helpers.h

Fixes: 7a387bed47f7 ("scripts/bpf: teach bpf_helpers_doc.py to dump BPF helper definitions")
Signed-off-by: Andrii Nakryiko <andriin@fb.com>
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Link: https://lore.kernel.org/bpf/20191010042534.290562-1-andriin@fb.com
Acked-by: Gary Lin <glin@suse.com>
---
 scripts/bpf_helpers_doc.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/scripts/bpf_helpers_doc.py
+++ b/scripts/bpf_helpers_doc.py
@@ -418,7 +418,7 @@ class PrinterHelpers(Printer):
 
             'struct __sk_buff',
             'struct sk_msg_md',
-            'struct xpd_md',
+            'struct xdp_md',
     ]
     known_types = {
             '...',
